Resources/translations/FSiAdminSecurity.pl.yml

```diff
@@ -26,7 +26,7 @@ admin:
         email: Email
         button:
           send: Wyślij mi instrukcje
-      mail_sent: Instrukcja zmiany hasła została wysłana pod podany adres email
+      mail_sent_if_correct: Jeśli email, który podałeś był prawidłowy, wtedy powinieneś otrzymać instrukcję zmiany hasła.
```

Comment:
Jeśli podany email był prawidłowy, to powinieneś otrzymać na niego instrukcję zmiany hasła.